src/wireviz/DataClasses.py

```diff
@@ -7,6 +7,16 @@
 from wireviz import wv_colors
 
 
+@dataclass
+class Image:
+    src: str
+    scale: str = "false"
+    width: Optional[int] = None
+    height: Optional[int] = None
+    fixedsize: bool = False
```

Comment:
Shouldn't this be `Optional` as well?

Reply:
See https://github.com/formatc1702/WireViz/pull/153#discussion_r471052220